cron-setup/docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+version: '2'
+services:
+  ews-orgmode:
+    build:
+      context: .
+    image: ews-orgmode-cron
+    volumes:
+      - ../../.ews-orgmode:/data

```

Comment:
I guess it would be good to mention that in the README, this encodes your local directory structure.

Reply:
This is already documented in the README. Perhaps it is a bit unclear?

Reply:
Apologies, I overlooked that.